keylime-agent/src/payloads.rs

```diff
@@ -571,7 +572,6 @@ echo hello > test-output
 
         let result = payload_tx.send(PayloadMessage::Shutdown).await;
         assert!(result.is_ok());
-        drop(payload_tx);
```

Comment:
When I added this I suspected that somehow this could help prevent the deadlocks we see on our tests when measuring test coverage with `tarpaulin`. But it didn't and I forgot to remove in the end.